### ISSUE TITLE
Navigator Error

### ISSUE THREAD

**Issue by tmalas** [ContinuumIO/anaconda-issues#8065]
## Main error
The following errors occurred:
## Traceback
```
CustomValidationError: Parameter default_python = '' declared in <<merged>> is invalid.
default_python value '' not of the form '[23].[0-9]'
```
## System information
```
python: 3.6.1
language: en
os: Darwin;17.3.0;Darwin Kernel Version 17.3.0: Thu Nov  9 18:09:22 PST 2017; root:xnu-4570.31.3~1/RELEASE_X86_64;x86_64;i386
version: 1.6.2
platform: osx-64
qt: 5.6.2
pyqt: 5.6.0
conda: 4.4.4
```

**Comment by goanpeca** [comment]
@tmalas we are working on a fix for this, making a release this week.

**Comment by tmalas** [comment]
Thanks...

**Comment by maimaiti** [comment]
I got the same issue after I updated anaconda.  

is this problem fixed? any new release?

Thanks

**Comment by goanpeca** [comment]
The conda path release will include this fix (Right @kalefranz  :-p )

**Comment by kalefranz** [comment]
Yeah https://github.com/conda/conda/pull/6682 will be in conda 4.4.8.  If we could get a few more of the pending issues merged we could cut another 4.4 patch release :-p
